tools/exports.js

```diff
@@ -17,3 +17,4 @@ exports["string_template"] = string_template;
 exports["tokenizer"] = tokenizer;
 exports["is_identifier"] = is_identifier;
 exports["SymbolDef"] = SymbolDef;
+exports["EXPECT_DIRECTIVE"] = EXPECT_DIRECTIVE;
```

Comment:
(done by copying the regexp to the tests, and leave the other copy untested in the code base)